snet-ui/AirdropRegistration/index.tsx

```diff
@@ -324,6 +324,22 @@ export default function AirdropRegistration({
                     View Schedule
                   </Button>
                 </Box>
+                <Box
+                  sx={{ display: "flex", justifyContent: "center", mt: [2, 0] }}
+                >
+                  <Button
+                    variant="contained"
+                    color="secondary"
+                    onClick={onViewRules}
+                    sx={{
+                      textTransform: "capitalize",
+                      width: 170,
```

Comment:
@raamb can you test the responsiveness of the button, since we are hard-coded width of it

Reply:
@rajeevbbqq its the same for all the other elements. Also I just brought this back from an earlier checkin

Reply:
@raamb it's ok but can you check, so we don't need to revisit for style fixes.

Reply:
Let's fix the responsiveness on a separate PR